src/settings/mod.rs

```diff
@@ -38,22 +39,53 @@ pub mod types;
 
 const CARGO_PKG_VERSION: &str = env!("CARGO_PKG_VERSION");
 
+#[derive(Debug)]
+pub struct AllSettings {
+    pub cli: CliSettings,
+    pub lib: Settings,
```

Comment:
Idly wondering if this should be `LibSettings`, and `AllSettings` should be `Settings`.

Reply:
Eh, `AllSettings` or `CombinedSettings` is probably for the aggregate struct. Although I do wonder if `Settings` should be `LibSettings` or something.

Reply:
I don't think `CliSettings` belongs in the `ruff` crate at all ... my plan is to move it to `ruff_cli` in the future. So I think it makes sense to keep it as `Settings` since these ought to be the only settings in the library crate.